moment-timezone-utils.js

```diff
@@ -95,6 +95,17 @@
 		return out.join(' ');
 	}
 
+	function packIsDst(dst) {
+		var out = [],
+			i;
+
+		for (i = 0; i < dst.length; i++) {
+			out[i] = dst[i];
+		}
+
+		return out.join('');
+	}
+
```

Comment:
I wonder if there is more compact way to store isDst ? Isn't dst just a flag `0` or `1` ?  Eg take it as binary, prefix it with `1` and then convert it to an hex format or use this base60 converter. The current implementation increase file size of 2021e by 14%, there might be space for optimisation

Reply:
zdump man page says:

> Each line is followed by isdst=D where D is positive, zero, or negative depending on whether the given time is daylight saving time, standard time, or an unknown  time  type

so while currently it is only 0 and 1, in theory it could be any number, thus i didn't know how to decrease the size really ... but yes, we currently need one byte for each dst info ... that might be overkill :/

Reply:
Interesting problem, we could solve it any time later as long as we have corresponding packer/ unpacker. 
If I get it right isdst is a flag with 3 states saving, standard or unknown. That should be packable somehow.... An array of a 3 states flag...

Reply:
I found some time to pay around with a packing algorithm. see branch https://github.com/js-joda/js-joda/commits/isdst-pack. The 3 state approach compact ratio is about 1/3. But practically the unknown state is probably without use, the only thing you can do there is to show a non dst case, i guess. So the last commit contains a 2 state approach and compact about 18%.